mono/utils/atomic.c

```diff
@@ -582,5 +582,9 @@ InterlockedCompareExchange64(volatile gint64 *dest, gint64 exch, gint64 comp)
 }
 
 #endif
+#endif
 
+#if defined(HOST_WIN32) && defined(_MSC_VER)
```

Comment:
ok let's see if I can find a nice home for it. Since there are other PR's with similar construct it might be more effective to merge first and then do a final PR changing this in all places at the same time.